### PR TITLE
3496 - GEO -> Update layers selection style 

### PR DESCRIPTION
**Pull request by minotogna**
https://github.com/openforis/fra-platform/assets/1138156/eb3729fe-ad71-41a4-9387-4687dabdc7ff

### CODE REVIEW

File: src/client/components/Inputs/SelectPrimary/SelectPrimary.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import Select from 'client/components/Inputs/Select'
+import { SelectProps } from 'client/components/Inputs/Select/types'
+
+const SelectPrimary: React.FC<SelectProps> = (props) => {
+  const { disabled, isClearable, isMulti, onChange, options, value } = props
+
+  return (
+    <Select
+      classNames={{ container: classNames('select-primary__container', { inverse: !value }) }}
```
Comment (yaguzmang) [inline src/client/components/Inputs/SelectPrimary/SelectPrimary.tsx:14]: This check `inverse: !value` would incorrectly return false if the Select has `isMulti` and the `value` array is empty. Should we account for that too using `Objects.isEmpty(value)` instead?

File: src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelControl.tsx [hunk 2]
```diff
@@ -1,13 +1,14 @@
-import './AgreementLevelControl.scss'
+import 'client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelControl.scss'
```
Comment (yaguzmang) [inline src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelControl.tsx:1]: Should this be `import './AgreementLevelControl.scss'`?

File: src/client/pages/Geo/Map/hooks/styles.ts [hunk 3]
```diff
@@ -0,0 +1,259 @@
+export const styles = [
+  {
+    elementType: 'geometry',
+    stylers: [
+      {
+        // color: '#20292f',
```
Comment (yaguzmang) [inline src/client/pages/Geo/Map/hooks/styles.ts]: Is this color comment for a reference?

File: src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelSelector/utils.ts [hunk 4]
```diff
@@ -3,23 +3,22 @@ import { CSSProperties } from 'react'
...
-    levelOffset >= 0 && level <= selectedLayersCount && levelOffset < eligiblePalette.length
+  // const isLevelSelectableAbove =
+  //   levelOffset >= 0 && level <= selectedLayersCount && levelOffset < eligiblePalette.length
 
   const style: CSSProperties = {}
 
-  if (isLevelSelectableAbove && eligiblePalette) {
-    style.backgroundColor = eligiblePalette[levelOffset]
-  }
+  // if (isLevelSelectableAbove && eligiblePalette) {
```
Comment (yaguzmang) [inline src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelSelector/utils.ts:17]: Should we remove these comments?

File: src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelSelector/hooks/useAgreementLevelData.ts [hunk 5]
```diff
@@ -29,7 +29,8 @@ export const useAgreementLevelSelectorData = (props: Props): Returned => {
   return useMemo<Returned>(() => {
     let eligiblePalette: Array<string> = []
     if (agreementPalette && currentSelectedLevel !== undefined) {
-      eligiblePalette = agreementPalette.slice(currentSelectedLevel - 1, selectedLayersCount)
+      // eligiblePalette = agreementPalette.slice(currentSelectedLevel - 1, selectedLayersCount)
```
Comment (yaguzmang) [inline src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelSelector/hooks/useAgreementLevelData.ts:32]: Should we remove this comment?

**Review by yaguzmang — APPROVED** [review]

File: src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelControl.tsx (hunk 2 above)
Comment (minotogna) [reply src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelControl.tsx:1]: 👍 coming in the next pr

File: src/client/pages/Geo/Map/hooks/styles.ts (hunk 3 above)
Comment (minotogna) [reply src/client/pages/Geo/Map/hooks/styles.ts]: 👍 coming in the next pr

File: src/client/components/Inputs/SelectPrimary/SelectPrimary.tsx (hunk 1 above)
Comment (minotogna) [reply src/client/components/Inputs/SelectPrimary/SelectPrimary.tsx:14]: 👍 coming in the next pr

File: src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelSelector/hooks/useAgreementLevelData.ts (hunk 5 above)
Comment (minotogna) [reply src/client/components/Navigation/NavGeo/Layer/AgreementLevelControl/AgreementLevelSelector/hooks/useAgreementLevelData.ts:32]: 👍 coming in the next pr

**Pull request merged**